src/components/views/auth/VectorAuthHeaderLogo.js

```diff
@@ -29,13 +29,15 @@ export default class VectorAuthHeaderLogo extends React.PureComponent {
     render() {
         const brandingConfig = SdkConfig.get().branding;
         let logoUrl = "themes/element/img/logos/element-logo.svg";
+        let logoUrlSecondaryVertical = "themes/element/img/logos/element-logo-secondary-vertical.png";
```

Comment:
Based on your conditional code below, I assume this meant to come from the config.json file?

Reply:
Yes if file exist then only we will load logo otherwise element's default wont be affected.

Reply:
Is
```
{logoUrlSecondaryVertical && 
```
a file exists check?

Reply:
yes.